src/ast.rs

```diff
@@ -243,6 +292,35 @@ impl Expr {
             OrExpr { span, .. } => span,
         }
     }
+
+    pub fn eidx(&self) -> u32 {
+        use Expr::*;
+        match self {
+            String { eidx, .. }
+            | RawString { eidx, .. }
+            | Number { eidx, .. }
+            | Bool { eidx, .. }
+            | Null { eidx, .. }
+            | Var { eidx, .. }
```

Comment:
>Var { eid [](http://example.com/codeflow?start=14&length=9)

very nit: can you order this and the fn above the same way? Just a few lines are out of order var bool null